ext/dba/tests/gh16390.phpt

```diff
@@ -2,6 +2,11 @@
 GH-16390 (dba_open() can segfault for "pathless" streams)
 --EXTENSIONS--
 dba
+--SKIPIF--
+<?php
+require_once __DIR__ . '/setup/setup_dba_tests.inc';
+check_skip('inifile');
```

Comment:
Comparing this with the SKIPIF section of https://github.com/php/php-src/pull/17005/files#diff-0f4f5dbb2a00f5b3a7c047f212cb553bbf1e3112ebe500f52b2188222a746260R9-R10, I wonder what's the preferred way nowadays.

Reply:
Ha, this happens every time I add a `SKIPIF`. They both get the job done, but the `check_skip()` function is a bit newer so I placed my bet on that.